### PR TITLE
feat(init): Create module

### PR DESCRIPTION
**Pull request by fbeuvens**
Add base structure, resource methods and testing

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+export const pollMessage = params => new BPromise((resolve, reject) => {
+  log.debug('SQS_POLL_MESSAGE', null, params);
+  receiveMessage(params)
+    .then(messages => {
+      log.debug('SQS_POLL_MESSAGE_SUCCESS', messages, params);
+      if (messages) {
+        resolve(messages);
+      } else {
+        log.debug('SQS_POLL_MESSAGE_THROTTLE', { throttling: params.throttling || 1000 }, params);
```
Comment (SagaStalin) [inline src/index.js]: MINOR Split this 98 characters long line (which is greater than 80 authorized). [rule](https://sonar.sagacify.com/coding_rules#rule_key=javascript%3ALineLength)

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        setTimeout(() => pollMessage(params).then(resolve).catch(reject), params.throttling);
```
Comment (SagaStalin) [inline src/index.js]: MINOR Split this 93 characters long line (which is greater than 80 authorized). [rule](https://sonar.sagacify.com/coding_rules#rule_key=javascript%3ALineLength)

**Comment by SagaStalin** [comment]
SonarQube analysis reported 2 issues
* MINOR 2 minor

Watch the comments in this conversation to review them.

**Comment by satblip** [comment]
LGTM

**Comment by fbeuvens** [comment]
Thanks louis le plus beau!

**Comment by fbeuvens** [comment]
Thanks again, it was so great!

**Pull request merged**